.github/workflows/latest-dependencies.yaml

```diff
@@ -44,36 +44,45 @@ jobs:
             runner_os: macos-latest
             default_target: true
             runnable: true
-          - target: x86_64-unknown-linux-gnu
-            runner_os: ubuntu-latest
+          - target: x86_64-pc-windows-msvc
+            runner_os: windows-latest
             default_target: true
             runnable: true
-          - target: x86_64-unknown-linux-musl
+          - target: x86_64-unknown-linux-gnu
             runner_os: ubuntu-latest
-            default_target: false
-            runnable: true
-          - target: x86_64-pc-windows-msvc
-            runner_os: windows-latest
             default_target: true
             runnable: true
+          # TODO: Enable Linux musl build again when upstream issue has been
```

Comment:
Could we re-enable this now that the workaround has been added to open62541-sys?